packages/react/src/React.mli

```diff
@@ -632,6 +632,22 @@ val useCallback5 : 'a -> 'b -> 'a
 val useCallback6 : 'a -> 'b -> 'a
 val useId : unit -> string
 
+type ('input, 'output) callback = 'input -> 'output
+
+val useSyncExternalStore :
+  subscribe:((unit -> unit) -> (unit, unit) callback) ->
+  getSnapshot:(unit -> 'snapshot) ->
+  'snapshot
+[@@deprecated
+  "Use useSyncExternalStoreWithServer instead. More info at \
```

Comment:
I'm getting very picky, but just inline their comment no?

```
'Missing getServerSnapshot, which is required for server-rendered content.',
```

Reply:
> I'm getting very picky
Not a problem to me 😁